components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryClientException.java

```diff
@@ -41,4 +43,13 @@ public IdentityRecoveryClientException(String errorCode, String message, Throwab
         super(errorCode, message, throwable);
     }
 
+    public IdentityRecoveryClientException(String errorCode, String message, String description, Throwable throwable) {
```

Comment:
If this is formatted there should be a new line

Reply:
Fixed with d8a7e0f199ca0dd41ddeb24807f49e89527d6b61